tools/cldr-apps/src/main/webapp/js/new/cldrSurvey.js

```diff
@@ -4325,32 +4245,95 @@ const cldrSurvey = (function () {
       });
   }
 
+  function setShower(id, func) {
+    showers[id] = func;
+  }
+
+  /**
+   * Add to the radio button, a more button style
+   *
+   * @param button
+   * @returns a newly created label element
+   */
+  function wrapRadio(button) {
+    var label = document.createElement("label");
+    label.title = "Vote";
+    label.className = "btn btn-default";
+    label.appendChild(button);
+    $(label).tooltip();
+    return label;
+  }
+
   /*
    * Make only these functions accessible from other files:
    */
   return {
-    updateIf: updateIf,
-    isReport: isReport,
+    INHERITANCE_MARKER: INHERITANCE_MARKER,
     addClass: addClass,
-    removeClass: removeClass,
-    removeAllChildNodes: removeAllChildNodes,
-    setDisplayed: setDisplayed,
-    isInputBusy: isInputBusy,
-    createChunk: createChunk,
+    addIcon: addIcon,
```

Comment:
FWIW you can write this as:

```js
return {
 INHERITANCE_MARKER,
 addClass,
 addIcon,
…
}
```

If the variable and the hash key have the exact same name, the variable doesn't have to be repeated.

Reply:
I didn't know that. Very cool improvement in ES6! The IIFE will be so much less clunky without the duplication. I've been picturing that we'll use "export" instead of IIFE, but who knows maybe some combination...